CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ _None_
 
 ### Bug Fixes
 
-_None_
+- `create_release_backmerge_pull_request` now deletes existing intermediate branches before creating them anew. [#601]
```

Comment:
Note that even if the behavior of the action **_changed_** for the case when a backmerge PR (and intermediate branch) already exists (previously errored, now delete the bramch + close the PR and create it anew), I still considered this a Bug Fix rather than a Breaking Change.

Reply:
Fair.